src/Dialect/ONNX/DialectBuilder.cpp

```diff
@@ -62,17 +72,26 @@ Value OnnxBuilder::matmul(Type Y, Value A, Value B, bool useGemm) const {
   return b.create<ONNXMatMulOp>(loc, Y, A, B);
 }
 
+Value OnnxBuilder::min(ValueRange inputs) const {
+  assert(inputs.size() >= 2 && "Expect at least two inputs");
+  return b.create<ONNXMinOp>(loc, inputs[0].getType(), inputs);
+}
+
+Value OnnxBuilder::mul(Value A, Value B) const {
+  return b.create<ONNXMulOp>(loc, A, B);
```

Comment:
I think adding an assertion (here and for other binary operators) to check that the input have the same data type would be useful.

Reply:
Done. Added an assertion for all binary operators.